addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -227,6 +269,20 @@ def _rtc_join_call(self, check_rtc_session_ids=None):
                 res['invitedMembers'] = [('ADD', list(invited_members._discuss_channel_member_format(fields={'id': True, 'channel': {}, 'persona': {'partner': {'id', 'name', 'im_status'}, 'guest': {'id', 'name', 'im_status'}}}).values()))]
         return res
 
+    def _get_rtc_server_info(self, rtc_session):
+        sfu_channel_uuid = self.channel_id.sfu_channel_uuid
+        sfu_server_url = self.channel_id.sfu_server_url
+        key = discuss.get_sfu_key(self.env)
```

Comment:
This implies that different sfu server that serve the same channel(s) must have the same key?

Reply:
There shouldn't be multiple SFU servers serving the same channel at the same time, if we want to use another server (with a different key), both the url and the key must be changed in the config parameters